test/translator/inquiry_test.cc

```diff
@@ -76,24 +76,29 @@ TEST_F(InquiryTest, InquiryToNvme) {
   uint32_t alloc_len;
   translator::Allocation allocations[2] = {{}};
 
-  translator::StatusCode status = translator::InquiryToNvme(
-      scsi_cmd_, nvme_cmds_[1], nvme_cmds_[0], alloc_len, nsid, allocations);
+  translator::StatusCode status =
+      translator::InquiryToNvme(scsi_cmd_, nvme_wrappers_[0], nvme_wrappers_[1],
```

Comment:
Change in ordering is deliberate, see PR description https://github.com/googleinterns/scsi2nvme/pull/164#issue-460531001